### ISSUE TITLE
Training performance and configuration

### ISSUE THREAD

**Issue by manigot** [xkaple00/deep-reinforcement-learning-for-crypto-trading#1]
I was able to run training successfully. I've rented an instance with 2x RTX 3090 on "Vast.ai". The problem is training performance is significantly lower than yours. On default config file num_rollout_workers is set to 15, num_envs_per_worker is set to 8 and rollout_fragment_length is set to 168. Therefore number of environment steps trained per trial is 15 * 8 * 168 equal to 20160. Whereas your tensorboard logs show that steps per trial is nearly a million. I had used different parameters for resources such as setting num_gpus_per_worker , num_cpus_per_worker etc. Also used another instance with single GPU. But couldn't achieve anything. Sorry, I'm new to rlllib. Could you elaborate your training config and process,please? Great work by the way, thank you.

**Comment by mostafaac30** [comment]
Is the ‘close_0060_FTM’ price column correct? I tested it, but the ‘close_0060_FTM’ column contains the same value throughout, and when I split the data, the chart shows straight lines.
@manigot

**Comment by dfoggy00** [comment]
I observed the same behavior on my end. Were you able to root cause the
issue?

**Comment by mostafaac30** [comment]
Still investigating the issue and trying to fix it. If interested, we can collaborate to run the model. My telegram: https://t.me/Mosta_fa009

**Comment by dfoggy00** [comment]
Yes, I'm interested in collaborating. It appears potentially all of the
columns scraped from binance have the same data from top to bottom. So I
would say the problem might be the binance scraper. Looking into it now.
I'll message you on telegram.

**Comment by biiiipy** [comment]
Did anyone was able to run the training normally? I too see the performance 50-100x slower than in the authors logs and the gpu is being fully utilized.

**Comment by denoiser-ml** [comment]
@manigot , @mostafaac30 , @dfoggy00, @biiiipy you are sharing interesting experience!
When you get data from Santiment with scraper_santiment.py , do you get only last one year for 1h resolution?
I think this is a problem because I am getting lot of Nan:

![Image](https://github.com/user-attachments/assets/f520a592-d673-4b86-9b60-2da7ac11a596)
